### PR TITLE
Fix config path in laravel docs

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Update path in laravel docs from `app/config/auth.php` to `config/auth.php`
Closes: #5367

### CODE REVIEW

**Comment by albertoperdomo** [comment]
Hello @joshcanhelp,
can you please review this PR when you have some time? Thanks!

File: articles/quickstart/backend/laravel/01-authorization.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ This is adequate for basic testing or if you don't have a requirement to persist
 Configure the `driver` in `/config/auth.php` to use `auth0`.
 
 ```php
-// app/config/auth.php
+// /config/auth.php
```
Comment (joshcanhelp) [inline articles/quickstart/backend/laravel/01-authorization.md]: Remove preceding slash

File: articles/quickstart/backend/laravel/01-authorization.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ This is adequate for basic testing or if you don't have a requirement to persist
 Configure the `driver` in `/config/auth.php` to use `auth0`.
```
Comment (joshcanhelp) [inline articles/quickstart/backend/laravel/01-authorization.md]: Remove preceding slash

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
Thanks @arshadkazmi42! Quick comment here then happy to merge!

**Review by joshcanhelp — APPROVED** [review]

**Comment by joshcanhelp** [comment]
@albertoperdomo - GTG!

**Comment by arshadkazmi42** [comment]
@joshcanhelp Thank you for the review 👍

**Comment by albertoperdomo** [comment]
Thanks @arshadkazmi42 and @joshcanhelp ! :)

**Pull request merged**